CMakeLists.txt

```diff
@@ -386,6 +386,16 @@ endif ()
 # Turns on all external libs like s3, kafka, ODBC, ...
 option(ENABLE_LIBRARIES "Enable all external libraries by default" ON)
 
+option(USE_PRECISE_AS_READ_FLOAT_TEXT_IMPL "Use precise as implementation method for reading floating point numbers from decimal representation of text" OFF)
```

Comment:
These CMake options look too specific to be useful for more than a handful of people.

If we really want to distinguish "simple" and "precise" float reading, then we should rather have a runtime setting for that.